Packs/rasterize/Integrations/rasterize/rasterize.py

```diff
@@ -343,17 +349,18 @@ def start_chrome_headless(chrome_port, chrome_binary=CHROME_EXE, user_options=""
         demisto.debug(f'Chrome started on port {chrome_port}, pid: {process.pid},returncode: {process.returncode}')
 
         if process:
-            CHROME_PROCESS = process
             demisto.debug(f'New Chrome session active on Port {chrome_port}')
             # Allow Chrome to initialize
             time.sleep(DEFAULT_RETRY_WAIT_IN_SECONDS)  # pylint: disable=E9003
             browser = is_chrome_running_locally(chrome_port)
             if browser:
-                write_info_file(PORT_FILE_PATH, chrome_port)
+                demisto.log(f"Created new browser at port {chrome_port}")  # TODO: Remove log
```

Comment:
Change to `demisto.debug` or `demisto.info` (in all places.)
```suggestion
                demisto.info(f"Created new browser at port {chrome_port}")  # TODO: Remove log
```

Reply:
Regarding the log's, I will check where it can be converted to debug, and where it is not important and it is more for my private tests, and I will delete these when the pr is confirmed.